### DISCUSSION TITLE
Show combinations in product list (back office) and disable/enable combinations

### DISCUSSION THREAD

**Discussion by espamoto** [PrestaShop/PrestaShop#40179] [Ideas]
#### Is your feature request related to a problem?

Hello, we want a feature that we can't find in Prestashop and we think that is essential. It's to show the combinations of the products in the catalog (product list in back office) because then we can see if a product has qty X and also the possibility to enable and disable the combinations in the back office.
We work with thousand of products that are backorders and we have to check the status all the weeks and enable or disable depending of the brand/distributor.
Thanks

[a comment by an automated account is not shown]

**Comment by khouloudbelguith** [comment]
Hi @espamoto,

Thanks for your report.
Ping @PrestaShop/product-team what do you think of this improvement?

Thanks!

**Comment by LouiseBonnard** [comment]
Hi @khouloudbelguith, @espamoto, thanks for this issue. If I understand it well, you would like to have like the 'Combinations' tab of the product page in the products listing, right?

<img width="950" alt="Capture d’écran 2020-10-19 à 11 45 52" src="https://user-images.githubusercontent.com/32565890/96429238-03072800-1201-11eb-824f-0d13b31e0a57.png">

**Comment by espamoto** [comment]
Hello, yes, I want to have all the combinations (size S ,M ,L...) in the same product listing where are all the products, and also be able to disable each combination.

**Comment by MatShir** [comment]
Hi @espamoto, 
Does the value of the stock on the product page still correspond to the actual stock?

Besides, if you know in advance the combinations to be deactivated you can thank to the import you can enter on an excel file the products to be deactivated in one go.
